packages/aws-client/tests/test-Lambda.js

```diff
@@ -15,7 +15,7 @@ test.serial('create, invoke and delete function', async (t) => {
       ZipFile: fs.readFileSync(require.resolve('@cumulus/test-data/fake-lambdas/hello.zip')),
     },
     FunctionName: functionName,
-    Role: randomString(),
+    Role: `arn:aws:iam::123456789012:role/${randomString()}`,
```

Comment:
role arn most in the correct form.

Reply:
Did this run afoul of any git commit security hooks?

Reply:
No, and 123456789012 is used in many other places.